src/generate/index.js

```diff
@@ -52,13 +46,11 @@ function generateContributorsList(options, contributors) {
     }),
     _.chunk(options.contributorsPerLine),
     _.map(formatLine),
-    function insertColumns(lines) {
-      const columnLine = createColumnLine(options, contributors)
-      return injectContentBetween(lines, columnLine, 1, 1)
-    },
-    _.join('\n'),
+    _.join('</tr><tr>'),
     newContent => {
-      return `\n${newContent}\n`
+      const style =
+        '<style>#emoji-table, #emoji-table td { border: 1px solid #ccc; }</style>'
+      return `\n${style}<table id="emoji-table" cellspacing=0 cellpadding=1><tr>${newContent}</tr></table>\n`
```

Comment:
[minor] it may be better to quote the attribute `cellspacing="0"`

> When in doubt, it’s probably best to just use quotes
https://mathiasbynens.be/notes/unquoted-attribute-values